src/common.ts

```diff
@@ -1,11 +1,37 @@
-export interface TransactionOptions {
-  readOnly?: boolean;
-  timeout?: number;
-  exactStaleness?: number;
-  readTimestamp?: Date;
-  returnTimestamp?: boolean;
-  strong?: boolean;
+import {google as spTypes} from '../proto/spanner';
+import {Operation as GaxOperation} from 'google-gax/build/src/longrunning';
+import {ServiceError} from 'grpc';
+
+//tslint:disable-next-line no-any
+export type Any = any;
```

Comment:
Any chance we could get rid of this? I just noticed something similar in BigQuery and it turned out that an any alias was being used all over the place in places it shouldn't. While it is annoying to add tslint comments, I actually kind of prefer the explicitness of doing so.

Reply:
Yeah, I've been starting to prefer the same. Will do 👌